src/components/bs5/quickexit/quickexit.scss

```diff
@@ -197,66 +200,47 @@
       }
     }
   }
-}
-.icon-info {
-  position: absolute;
-  inset-inline-start: calc(var(--#{$prefix}quick-exit-spacing) * 2.5);
-  background-repeat: no-repeat;
-  margin-block-start: 4px;
-  background-image: url("data:image/svg+xml,<svg width='18' height='19' viewBox='0 0 18 19' fill='none' xmlns='http://www.w3.org/2000/svg'><path d='M9 0.25C4.00781 0.25 0 4.29297 0 9.25C0 14.2422 4.00781 18.25 9 18.25C13.957 18.25 18 14.2422 18 9.25C18 4.29297 13.957 0.25 9 0.25ZM9 17.125C4.64062 17.125 1.125 13.6094 1.125 9.25C1.125 4.92578 4.64062 1.375 9 1.375C13.3242 1.375 16.875 4.92578 16.875 9.25C16.875 13.6094 13.3242 17.125 9 17.125ZM9 6.71875C9.45703 6.71875 9.84375 6.36719 9.84375 5.875C9.84375 5.41797 9.45703 5.03125 9 5.03125C8.50781 5.03125 8.15625 5.41797 8.15625 5.875C8.15625 6.36719 8.50781 6.71875 9 6.71875ZM10.6875 12.625H9.5625V8.6875C9.5625 8.40625 9.28125 8.125 9 8.125H7.875C7.55859 8.125 7.3125 8.40625 7.3125 8.6875C7.3125 9.00391 7.55859 9.25 7.875 9.25H8.4375V12.625H7.3125C6.99609 12.625 6.75 12.9062 6.75 13.1875C6.75 13.5039 6.99609 13.75 7.3125 13.75H10.6875C10.9688 13.75 11.25 13.5039 11.25 13.1875C11.25 12.9062 10.9688 12.625 10.6875 12.625Z' fill='white'/></svg>");
-}
-.qld-quick-exit-inner.has-tooltip {
-  .qg-quick-exit__tips {
-    padding-inline-start: 0;
-  }
-  @media screen and (max-width: 925px) {
-    .not-mobile {
-      display: none;
+  .qld-quick-exit-inner.has-tooltip {
+    .qg-quick-exit__tips {
+      padding-inline-start: 0;
+    }
+    @media screen and (max-width: 925px) {
```

Comment:
Similar to my point above in using @include media-breakpoint for consistency.
Since **925px** is not included in predefined on QDGS `$grid-breakpoints` variable, I would check with the designer if this is a new breakpoint or an edge case.

Totally understand if time is tight here.

Reply:
@elvishu thanks for that.
I have updated the css with the @include media-breakpoint